addons/CLib/Core/Misc/fn_cachedCall.sqf

```diff
@@ -21,7 +21,8 @@
     0: Return Name <TYPE>
 */
 params ["_uid", "_fnc", "_args", "_duration", "_event"];
-if (([GVAR(cachedCall), _uid, [-9999999]] call CFUNC(getVariable)) select 0 < time) then {
+private _lastTime = (GVAR(cachedCall) getVariable [_uid, [-999999999]]) select 0;
```

Comment:
This is not the "last" time. In most cases in a timestamp in the future so I would name it differently.